### PR TITLE
dynamic adding and removal of books done

### PR DESCRIPTION
**Pull request by muhammaduzairrazaq**
**I have fulfilled the following requirements:**

- Linters added.
- Dynamic adding and removal of books completed.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+const booksobjects = parseString();
+
+const displayBooks = () => {
+  let bookData = '';
+  for (let i = 0; i < booksobjects.length; i += 1) {
+    bookData += `<p>${booksobjects[i].bookTitle}</p>
+        <p>${booksobjects[i].bookAuthor}</p>
+        <button class="remove-book" onclick="removeBook(${i})">Remove</button>
+        <hr>`;
+  }
```
Comment (emhamza) [inline script.js]: Hi @muhammaduzairrazaq 🔢 
Good job so far but I am afraid your onClick event is not working properly. When I tried to add a book in you app the Add button is not working at all. Please fix the potential bug:
bug

**Review by emhamza — CHANGES_REQUESTED** [review]
Hi @muhammaduzairrazaq 
Good job so far but there are some required changes that you need to work on in order to move on to the next part! 🔢 🏁

**Comment by muhammaduzairrazaq** [comment]
**Thanks for reviewing issue resolved :sunglasses: **
image

**Review by emhamza — APPROVED** [review]
Hi 👍 @muhammaduzairrazaq 
Congratulations on this accomplishment. There is nothing to say other than your PR is approved. Its time to marge it. 
🥳

**Pull request merged**